docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -378,6 +378,317 @@ and a file system location of ``/var/my_tables/test_table``::
         partitioning = ARRAY['c1', 'c2'],
         location = '/var/my_tables/test_table')
 
+.. _iceberg-metadata-tables:
+
+Metadata tables
+---------------
+
+The connector exposes for each Iceberg table several metadata tables
```

Comment:
The connector exposes several metadata tables for each Iceberg table. These metadata table contain information about the internal structure of the Iceberg table.